src/azul/plugins/metadata/hca/transform.py

```diff
@@ -274,6 +274,23 @@ def from_index(self, value: str) -> Optional[JSON]:
 value_and_unit: ValueAndUnit = ValueAndUnit()
 
 
+@attr.s(frozen=True, auto_attribs=True, kw_only=True)
+class Nested(PassThrough[JSON]):
+    es_type = 'nested'
+    properties: Mapping[str, FieldType]
+
+    # This allows for some level of interoperability between field types that
+    # are Nested vs Dictionaries.
```

Comment:
```suggestion
    # This allows for some level of interoperability between fields that
    # are Nested vs plain dictionaries.
```

 Is this really necessary? I see only one call site and it is guarded by an instance check for Nested.

Reply:
Actually, I see no new call site so I may be wrong. Please discuss with me in PL.

Reply:
self note: PFB Manifest generation